examples/examples_test.go

```diff
@@ -66,6 +66,23 @@ func TestAws(t *testing.T) {
 	}
 }
 
+func TestBrokenDockerfile(t *testing.T) {
+	cwd, err := os.Getwd()
+	if !assert.NoError(t, err) {
+		t.FailNow()
+	}
+
+	opts := base.With(integration.ProgramTestOptions{
+		Dependencies: []string{
+			"@pulumi/docker",
+		},
+		Dir: path.Join(cwd, "broken_dockerfile"),
+		ExpectFailure: true,
+		SkipRefresh: true,
```

Comment:
note: i will be able to pass in SkipUpdate here once https://github.com/pulumi/pulumi/pull/3894 goes in.  This will allow us to validate that it's the preview step that is causing us to fail.